### DISCUSSION TITLE
4.3 Update

### DISCUSSION THREAD

**Discussion by dgasp24** [jagrosh/MusicBot#1635] [Questions] [answered]
Can anyone answer when 4.3 will be released since it'll hopefully fix the youtube error (403)? Thank you!

**Comment by jagrosh** [comment, marked as answer]
Soon™

**Reply by dgasp24** [reply]
Like within a week or so? Sorry if I'm asking too much, but still thank you!

**Reply by MingDaMonger** [reply]
Im pretty sure the musicbot is dependent on lavaplayer, so until lavaplayer updates jagrosh is at a roadblock. every time youtube updates how they handle videos lavaplayer has a seizure and breaks

**Comment by DEAMJAVA** [comment]
i was having the same question, last time it was fixed within a day but this time it is taking a while

**Comment by DEAMJAVA** [comment]
my solution, i created my own temporary music bot till this gets fix lol

**Reply by Starsseeker2078** [reply]
How do you create your own music bot ?

**Reply by DEAMJAVA** [reply]
using python (i am also a developer)

**Reply by Jorgef231** [reply]
how could I make a bot myself? I'm not a developer and don't know how to use python

**Reply by DEAMJAVA** [reply]
i think you got the term "created my own bot" wrong, by that i meant that i developed my own bot code

**Reply by juab888a** [reply]
would you mind sharing?
